opt-tools/docker/docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: '3'
+
+services:
+
+  postgres_db:
+    image: postgres
+    restart: always
+
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=school
+      - POSTGRES_DB=school
+
+    volumes:
+      - ../../data/postgres:/var/lib/postgresql/data
+      - ../../data/dump:/db-dump
+
+    ports:
+      - "5432:5432"
+
+    networks:
+      - docker-practice-net
+
+
+  redis:
+    image: "redis:alpine"
```

Comment:
One suggestion: the best practice is to set docker image version explicitly, in our case it might be: _maven:3.6.3-jdk-11_
One question: Why do we need double quotes here?

Reply:
I used double quotes here because the example that I found was with double quotes.

Reply:
OK, I got it. Well, it should works without double quotes. It confuses when rest images are without double quotes.